lib/viewing/Layout.js

```diff
@@ -45,7 +49,7 @@ var Layout = View.extend({
     },
 
     reattach: function() {
-        this.setElement($("html").get(0));
+        this.setElement($("html").eq(0));
```

Comment:
Yes, but would like to select the first one in the event there are other html tags on the page, e.g. within an iframe. Am not sure if $("html") would grab those ones as well of the top of my head.